Jenkinsfile

```diff
@@ -34,10 +35,7 @@ properties([
     pipelineTriggers(prod ? [cron("H/30 * * * *")] : []),
     parameters([
       choice(name: 'STREAM',
-             // XXX: Just pretend we're the testing stream for now... in
-             // reality, we're closer to what "bodhi-updates" will be. Though the
-             // testing stream is the main stream.
-             choices: ['testing' /*, 'stable', 'testing-devel', 'bodhi-updates', etc... */ ],
+             choices: (streams.prod + streams.devel + streams.mechanical),
```

Comment:
if I just do `oc start-build dustymabe-fedora-coreos-pipeline` which choice is used?

Reply:
Unless you overrode it through `./devel-up`, you'd get `testing` (the first one). We can change that so it defaults to `testing-devel`. But really, if you care which specific one gets built (i.e. you're testing something sensitive to the FCOS config itself), you should use `./devel-up --config`.

Reply:
yeah let's switch it so that `testing-devel` is first (i.e. default behavior is the default branch in fcos-configs repo). I think that is a sane default we can live with.

Reply:
Done! :arrow_down: